api/client/src/test/resources/logback-test.xml

```diff
@@ -27,4 +27,6 @@
     <level value="${test.log.level:-INFO}"/>
     <appender-ref ref="console"/>
   </root>
+  <logger name="com.github.dockerjava" level="WARN"/>
+  <logger name="org.projectnessie.client.auth.oauth2" level="DEBUG"/>
```

Comment:
nit: Do we still need debug logs in STDOUT?

Reply:
Sorry that was a leftover.